offchain-modules/package.json

```
@@ -54,7 +54,8 @@
     "startDevDockerDeps": "docker-compose -f ../docker/docker-compose.dev.yml up -d",
     "destroyDevDockerDeps": "docker-compose -f ../docker/docker-compose.dev.yml down",
     "dev-docker:install": "docker run --rm -v $(pwd):/app -v force-bridge-node-modules:/app/node_modules node:14.18.1-bullseye bash -c 'cd /app && yarn build'",
-    "dev-docker:generate": "docker run --rm -e FORCE_BRIDGE_PROJECT_DIR=$(dirname \"$(pwd)\") -v $(dirname \"$(pwd)\"):/app -v force-bridge-node-modules:/app/offchain-modules/node_modules --network='docker_force-dev-net' node:14.18.1-bullseye bash -c 'cd /app/offchain-modules && npx ts-node packages/scripts/src/devDocker.ts'",
+    "dev-docker:generate": "docker run --rm -e FORCE_BRIDGE_PROJECT_DIR=$(dirname \"$(pwd)\") -v $(dirname \"$(pwd)\"):/app -v force-bridge-node-modules:/app/offchain-modules/node_modules --network='docker_force-dev-net' node:14.18.1-bullseye bash -c 'cd /app/offchain-modules && npx ts-node packages/scripts/src/devDocker.ts' && yarn dev-docker:ui",
```

```suggestion
    "dev-docker:generate": "docker run --rm -e FORCE_BRIDGE_PROJECT_DIR=$(dirname \"$(pwd)\") -v $(dirname \"$(pwd)\"):/app -v force-bridge-node-modules:/app/offchain-modules/node_modules --network='docker_force-dev-net' node:14.18.1-bullseye bash -c 'cd /app/offchain-modules && npx ts-node packages/scripts/src/devDocker.ts'",
```

If we want ui, we can run `yarn dev-docker:ui` separately